audit/README.md

```diff
@@ -21,6 +21,7 @@ Checked dig1wd02ktcvpananlvd9u6jm3x3ap3vmw59rhen8u with required allocation of 6
 ```digd unsafe-reset-all``` is required, all coin denomination in genesis.json must be changed from "stake" to "udig". Chain works well after that.
 
```

Comment:
Right here, please put exact instructions to make a single-node testnet.

Reply:
should that be a link to the guide or I just put it there?